client/src/routes/admin/roles/$role.tsx

```
@@ -0,0 +1,299 @@
+                  <AlertDialogTrigger
+                    key={user.id}
+                    asChild
+                    onMouseDown={() => {
+                      setSelectedUser(user);
+                      setDialogOpen(true);
+                    }}
+                  >
+                    <div
+                      className="hover:bg-accent flex cursor-pointer items-center space-x-2 rounded px-1"
+                      onMouseDown={() => console.log("test")}
+                    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)